config-object-store/src/main/java/org/hypertrace/config/objectstore/ConfigsResponseImpl.java

```diff
@@ -0,0 +1,24 @@
+package org.hypertrace.config.objectstore;
+
+import java.util.List;
+
+public class ConfigsResponseImpl<T> implements ConfigsResponse<T> {
```

Comment:
nit: when separating interface and impl, make the impl package private
```suggestion
class ConfigsResponseImpl<T> implements ConfigsResponse<T> {
```